docs/BindingsOverview.md

```diff
@@ -152,6 +152,12 @@ The delay in milliseconds between processing each batch of changes.
 
 The upper limit on the number of pending changes in the user table that are allowed per application-worker. If the count of changes exceeds this limit, it may result in a scale out. The setting only applies for Azure Function Apps with runtime driven scaling enabled. See the [Scaling](#scaling-for-trigger-bindings) section for more information.
 
+#### WEBSITE_SITE_NAME
+
```

Comment:
Can we also add this note:

"If they have 2 sql triggers with same functionName locally, not having WEBSITE_SITE_NAME would mean that the same checkpointing tables would be used for both triggers. So only one of the functions would be triggered. "